pyomo/contrib/doe/result.py

```diff
@@ -57,11 +56,12 @@ def __init__(
         """Analyze the FIM result for a single run
 
         Parameters
-        ----------
+        -----------
         para_name:
             A ``list`` of parameter names
         measure_object:
-            measurement information object
+            A measurement ``object`` which contains the Pyomo variable names and their corresponding indices and
+            bounds for experimental measurements
         jacobian_info:
             the jacobian for this measurement object
         all_jacobian_info:
```

Comment:
What is the overall Jacobian? What does this assume?

Reply:
When would we pass in both jacobian_info and all_jacobian_info?

Reply:
When we create result object in `DesignOfExperiments`, we are passing the full jacobian to result object. Full jacobian means all jacobian for every measurement variables. Then if we are given a subset of measurements, we are creating another result object, this object only has part of the jacobians (only those measurement variables in this subset).